src/itest/java/uk/gov/companieshouse/charges/data/repository/RepositoryITest.java

```diff
@@ -35,6 +46,30 @@ void should_return_mongodb_as_running() {
     Assertions.assertTrue(mongoDBContainer.isRunning());
   }
 
+  @Test
+  void should_save_and_retrieve_charges_data() {
+
+    ChargesDocument chargesDocument = createChargesDocument("CH253434", "jbvgV-Zu-i8bRkypE0AEJx1N_Sk");
+
+    chargesRepository.save(chargesDocument);
+
+    List<ChargesDocument> documents = chargesRepository.findAll();
+    assertThat(documents).hasSize(1);
+    assertThat(documents.get(0).getData().getId()).isEqualTo(chargesDocument.getData().getId());
+    assertThat(documents.get(0).getCompanyNumber()).isEqualTo(chargesDocument.getCompanyNumber());
+    assertThat(documents.get(0).getUpdated().getAt()).isEqualTo(chargesDocument.getUpdated().getAt());
+  }
+
+  private ChargesDocument createChargesDocument(String companyNumber, String chargeId) {
+
+    ChargeApi externalData = new ChargeApi();
+    externalData.setId(chargeId);
```

Comment:
ChargeAPI object can be de-serialised from a valid json file to make the test more comphrehensive